packages/blockchain-extension/test/commands/packageSmartContractCommand.test.ts

```diff
@@ -39,20 +39,23 @@ describe('packageSmartContract', () => {
     const typescriptPath: string = path.join(testWorkspace, 'typescriptProject');
     const invalidPath: string = path.join(testWorkspace, '  invalid package name! ');
     const golangPath: string = path.join(testWorkspace, 'src', 'goProject');
+    const golangPathFurther: string = path.join(testWorkspace, 'src', 'child', 'goProject');
     const wrongGolangPath: string = path.join(testWorkspace, 'goProject');
     const javaPath: string = path.join(testWorkspace, 'javaProject');
     const emptyContent: string = '{}';
 
     let folders: Array<any> = [];
 
-    async function createTestFiles(packageName: string, version: string, language: string, createValid: boolean, createMetadata: boolean, createWrongPlace: boolean = false): Promise<void> {
+    async function createTestFiles(packageName: string, version: string, language: string, createValid: boolean, createMetadata: boolean, createWrongPlace: boolean = false, srcChild: boolean = true): Promise<void> {
         let projectDir: string;
         if (language === 'golang') {
             if (createWrongPlace) {
                 folders[2].uri = vscode.Uri.file(wrongGolangPath);
                 projectDir = path.join(testWorkspace, packageName);
-            } else {
+            } else if (srcChild) {
                 projectDir = path.join(testWorkspace, 'src', packageName);
+            } else {
+                projectDir = path.join(testWorkspace, 'src', 'child', packageName);
```

Comment:
Just a suggestion  to change the folder name between src and packageName to something else. It becomes confusing when later on tests that mention something is not a child to then have child in the name. It's not wrong, just a bit confusing and makes understanding the tests harder.